docs/standard/native-interop/best-practices.md

```diff
@@ -84,6 +84,8 @@ GUIDs are usable directly in signatures. Many Windows APIs take `GUID&` type ali
 
 Blittable types are types that have the same bit-level representation in managed and native code. As such they do not need to be converted to another format to be marshaled to and from native code, and as this improves performance they should be preferred. Some types are not blittable but are known to contain blittable contents. These types have similar optimizations as blittable types when they are not contained in another type, but are not considered blittable when in fields of structs or for the purposes of [`UnmanagedCallersOnlyAttribute`](xref:System.Runtime.InteropServices.UnmanagedCallersOnlyAttribute).
 
+### Blittable types when runtime marshalling is enabled
+
 **Blittable types:**
 
 - `byte`, `sbyte`, `short`, `ushort`, `int`, `uint`, `long`, `ulong`, `single`, `double`
```

Comment:
This covers `string` below and I would assume is no longer supported when `runtime marshalling is disabled`. Is it worth explicitly calling that out?

Reply:
I missed the sentence that does cover this when reading the first time so looks like no action is really needed, other than maybe explicitly calling our `string`/`arrays` if desired.
> The concept of "types with blittable contents" does not apply when runtime marshalling is disabled.

👍